docs/glossary.md

```diff
@@ -0,0 +1,65 @@
+| Serving function         | Subtype of MLRun function that uses the Nuclio runtime specifically for serving ML models or real-time pipelines                                                                                     |
+| storey                   | Asynchronous streaming library for real time event processing and feature extraction. Used in Iguazio's feature store and real-time pipelines                                                |
+|                          |                                                                        
+
+## Iguazio (V3IO) Terms
+| Name                                       | Description          |   
+|--------------------------------------------------|---------------------------------------------------------------------------| 
+| Consumer group           | Set of consumers that cooperate to consume data from some topics                                                                                                                             |
+| Key Value (KV) store     | Type of storage where data is stored by a specific key, allows for real-time lookups                                                                                                         |
+| TSDB                     | Time series database: part of V3IO                                                                                                                                                           |
+| V3IO                     | Iguazio real-time data layer, supports several formats including KV, Block, File, Streams, and more                                                                                                    |
+| V3IO shard               | Uniquely identified data sets within a V3IO stream. Similar to a Kafka partition                                                                                                              |
+| V3IO stream              | Streaming mechanism part of Iguazio's V3IO data layer. Similar to a Kafka stream                                                                                                              |
+
+## Standard ML Terms
+| Name                                       | Description          |   
+|--------------------------------------------------|---------------------------------------------------------------------------| 
+| Artifact                 | The input or output of experiments and model training runs that  can be used in many runs across the project.                                                                                  |
+| DAG                      | Directed acyclic graph                                                                                                                                                                        |
```

Comment:
Directed acyclic graph, used to describe workflows/pipelines.

Reply:
updated